### ISSUE TITLE
The download zip shows an error when unzipping it.

### ISSUE THREAD

**Issue by windcr** [alterzero/RBPN-PyTorch#10]
The total size of the zip files is 1.01G. I download it for full size but something is wrong with unzipping it. How can I solve this problem?

**Comment by alterzero** [comment]
I don't know what file that you mentioned here.
For dataset, it is more than that. 
As far as I know Vimeo90K train-test set is roughly around 82GB.

**Comment by windcr** [comment]
Sorry, maybe my description is not clear. The zip files refer to the project files of RBPN-PyTorch.
